docs/preview/index.md

```diff
@@ -3,41 +3,52 @@ title: "Home"
 layout: default
 ---
 
+# Using our templates
+
+The `Arcus.Templates` has several build-in project templates available:
+* [Web API](features/web-api-template)
```

Comment:
Nit - Let's order this list alphabetically

Reply:
Done that. Resolving.